src/main/java/com/synopsys/integration/blackduck/nexus3/task/common/CommonTaskFilters.java

```diff
@@ -96,6 +96,9 @@ public boolean skipAssetProcessing(DateTime lastModified, String fullPathName, S
     }
 
     public boolean doesExtensionMatch(String filename, String allowedExtensions) {
+        if (StringUtils.isBlank(filename)) {
+            return false;
+        }
```

Comment:
Adding this to prevent us from scanning assets that have no filename, even if they provide a wildcard to scan everything.